app/models/family.rb

```diff
@@ -1683,8 +1686,46 @@ def none_applying_coverage?
     active_family_members.none?(&:is_applying_coverage)
   end
 
+  # Returns the most recently created determined application, whether it's an SBM or FAA application.
```

Comment:
This comment explains the behavior of the method, but not why we'd want it to behave that way.  Why would we distinguish between the two determined apps using the created_on field? Would the sbm_app always have an older created_at timestamp because it is created first in the user flow?

Reply:
So Financial Assistance Application contains all the info that Individual Market Application PLUS the info relevant to financial assistance.  If someone is new to the system and they opt to apply for assistance, they will begin with a Financial Assistance Application and not even have a Individual Market Application. (just noting the difference with how this works compared to current FAA implementation)